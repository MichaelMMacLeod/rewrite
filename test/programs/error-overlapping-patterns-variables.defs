(def ($n 0) _)
(def (0 $m) _)