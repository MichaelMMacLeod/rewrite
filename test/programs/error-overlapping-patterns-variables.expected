error[E007]: overlapping patterns
./test/programs/error-overlapping-patterns-variables.defs:1:6
   |
 1 | (def ($n 0) _)
   |      ^^^^^^ this pattern may match the same term as ...
./test/programs/error-overlapping-patterns-variables.defs:2:6
   |
 2 | (def (0 $m) _)
   |      ^^^^^^ ... this other pattern
help: patterns possibly matching the same term are not allowed
